sushichef.py

```diff
@@ -0,0 +1,355 @@
+        base_path = build_path([DATA_DIR, CHANNEL_SOURCE_ID])
+        videos_url_path = os.path.join(base_path, "{}.json".format(source_id_hash))
+
+        if file_exists(videos_url_path) and LOAD_VIDEO_LIST is True:
+            with open(videos_url_path, "r") as f:
+                name_url = json.load(f)
+        else:
+            for url in self.playlist_links():
+                youtube = YouTubeResourceNode(url)
+                info = youtube.get_resource_info()
+                name_url.append((info["title"], url))
+            with open(videos_url_path, "w") as f:
+                json.dump(name_url, f)
+        return name_url
+
+    def subtitles_dict(self):
+        subs = []
+        video_info = self.get_resource_subtitles()
+        if video_info is not None:
+            video_id = video_info["id"]
+            if 'subtitles' in video_info:
+                subtitles_info = video_info["subtitles"]
+                for language in subtitles_info.keys():
+                    subs.append(dict(file_type=SUBTITLES_FILE, youtube_id=video_id, language=language))
+        return subs
+
+    def download(self, download=True, base_path=None):
+        info = super(YouTubeResourceNode, self).download(base_path=base_path)
+        self.filepath = info["filename"]
+        self.title = info["title"]
+
+    def to_node(self):
+        if self.filepath is not None:
+            files = [dict(file_type=content_kinds.VIDEO, path=self.filepath)]
+            files += self.subtitles_dict()
+            node = dict(
+                kind=content_kinds.VIDEO,
+                source_id=self.source_id,
+                title=self.title,
+                description='',
+                author=AUTHOR,
+                files=files,
+                language=self.lang,
+                license=LICENSE
+            )
+            return node
+
+
+# The chef subclass
+################################################################################
+class ScienceAhmedChef(JsonTreeChef):
+    TREES_DATA_DIR = os.path.join(DATA_DIR, 'trees')
+
+    def __init__(self):
+        build_path([ScienceAhmedChef.TREES_DATA_DIR])
+        super(ScienceAhmedChef, self).__init__()
+
+    def pre_run(self, args, options):
+        channel_tree = self.scrape(args, options)
+        self.write_tree_to_json(channel_tree)
+
+    def lessons(self):
+        global CHANNEL_SOURCE_ID
+        self.RICECOOKER_JSON_TREE = 'ricecooker_json_tree_professional.json'
+        CHANNEL_NAME = "Sciences for Lower Secondary Learners (العربيّة)"
+        CHANNEL_SOURCE_ID = "sushi-chef-science-ahmed-al-hoot-ar"
+        channel_tree = dict(
+                source_domain=CHANNEL_DOMAIN,
+                source_id=CHANNEL_SOURCE_ID,
+                title=CHANNEL_NAME,
+                description="""Mr. Ahmad Al-Hoot from Egypt provides 13-16 year old learners with video lessons covered in several national curricula across the region in chemistry, biology and geology."""
```

Comment:
@elaeon
Could you please change the description to the Arabic text from here: https://docs.google.com/document/d/1PQi6y-A4ZYQOepUpe4lldK3gbquUSy3e9NLEGeIex8c/edit#bookmark=id.unqhypiqmiic

I see you already changed the title, so after that this one looks ready to go!